lib/assets/javascripts/libraries/map_reduce_utils.js

```diff
@@ -135,10 +135,14 @@ root.calculate = function(record, population, denominator, numerator, exclusion,
 
     }
   }
+  
+  if(variables){
+    variables();
+  }
```

Comment:
Looks like the "variables" variable isn't defined in the "calculate" method definition.